src/ensembl/src/content/app/species/components/species-stats/SpeciesStats.scss

```diff
@@ -7,38 +7,31 @@ $secondary-text-color: $dark-grey;
   margin-bottom: 30px;
   display: inline-block;
   vertical-align: top;
+  font-size: 13px;
```

Comment:
same here: default font size, inherited from the `html` element, doesn't change anything if disabled, so likely redundant

Reply:
Removed it